apps/experiments/models.py

```diff
@@ -639,8 +639,27 @@ def send_bot_message(self, instruction_prompt: str, fail_silently=True):
         bot_message = bot_prompt_for_user(self, prompt_instruction=instruction_prompt)
         try_send_message(self, message=bot_message, fail_silently=fail_silently)
 
+    def get_participant_scheduled_messages(self):
+        """Returns all scheduled messages for the associated participant for this session's experiment as well as
+        any child experiments in the case where the experiment is a parent"""
+        from apps.events.models import ScheduledMessage
+
+        child_experiments = ExperimentRoute.objects.filter(team=self.team, parent=self.experiment).values("child")
+        messages = ScheduledMessage.objects.filter(
+            Q(experiment=self.experiment) | Q(experiment__in=models.Subquery(child_experiments)),
+            participant=self.participant,
+            team=self.team,
+        )
```

Comment:
```suggestion
        ).select_related("action")
```

Reply:
Lol, I find it funny to see this since I had this in! I was debugging a query I tried to build and removed this part since it cluttered the sql....I guess my mind went too far and forgot to clean up after itself :rofl:. Thanks for spotting this